setup.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
-import re
+import os
 
 from setuptools import setup
 from setuptools.command.test import test
 
+from click_default_group import __version__ as version
 
-with open('click_default_group.py') as f:
-    version = re.search(r'__version__\s*=\s*\'(.+?)\'', f.read()).group(1)
-assert version
```

Comment:
Version can be easily imported, there is no need to parse the code to retrieve version.

Reply:
It looks like a bad idea. It makes a circular dependency between `setup.py` and `click_default_group`. Because `click_default_group` depends on `click` (by `import click`), `python setup.py` will fail with `ImportError` if the system has no `click` yet.

Reply:
You're right, I didn't see that.